modules/ROOT/pages/platform/security/encryption.adoc

```diff
@@ -11,46 +11,31 @@ By default, each cloud provider encrypts all backup buckets (including the objec
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-[NOTE]
-====
-This feature has been released as a public GA for AuraDB Enterprise and AuraDS Enterprise for AWS managed keys.
-GCP’s Cloud Key Management and Azure’s Key Vault are in development.
-====
-
 A Customer Managed Key (CMK) gives you more control over key operations than the standard Neo4j encryption.
 These are created and managed using a supported cloud key management service (KMS). 
 Externally, Customer Managed Keys are also known as Customer Managed Encryption Keys (CMEK).
 
 When using a Customer Managed Key, all data at rest is encrypted with the key.
 Customer Managed Keys are supported for v4.x and v5.x instances.
 
+Customer Managed Keys are designed to stay within the cloud provider’s KMS to maintain their security.
 When using Customer Managed Keys, you give Aura permission to encrypt and decrypt using the key, but Aura has no access to the key’s material.
 Aura has no control over the availability of your externally managed key in the KMS.
-If you lose keys that are managed outside of Aura, Aura can’t recover your data.
+If you lose keys that are managed outside of Aura in the cloud provider's KMS, you will not be able to access the data.
```

Comment:
I would remove the "that are managed outside of Aura" here. Its a bit confusing, since all keys are managed in the Cloud Provider KMS. So it would be
`If you lose keys in the cloud provider's KMS, you ...`